README.md

```diff
@@ -124,7 +125,6 @@ We thank the following people for their extensive assistance in the development
 - [cgpu](https://github.com/cgpu)
 - [gulfshores](https://github.com/gulfshores)
 - [pallolason](https://github.com/pallolason)
-- [Silvia Morini](https://github.com/silviamorins)
```

Comment:
Did we add the most recent contributors?

Reply:
added Oskar, Nick & Susanne;  Lasse & Anders were already there